aderyn_core/src/detect/mod.rs

```diff
@@ -1,4 +1,5 @@
 pub mod detector;
+pub(crate) mod experimental;
```

Comment:
This should be pub(crate) because the driver (and therefore nyth) shouldn't see this module